doc/pr2_tutorials/planning/src/move_group_interface_tutorial.cpp

```diff
@@ -177,7 +177,9 @@ int main(int argc, char **argv)
   // We will reuse the old goal that we had and plan to it.
   // Note that this will only work if the current state already 
   // satisfies the path constraints. So, we need to set the start
-  // state to a new pose. 
+  // state to a new pose. We also increase the default planning time from 5s to 60s.
+  
+  my_plan.planning_time_ = 60;
```

Comment:
From your testing was 60s sometimes used? That seems really really high. How about 30s so MoveIt! doesn't look as bad?